packages/gatsby-theme-newrelic/src/components/SearchInput.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+import Icon from './Icon';
+
+const SIZES = {
+  MEDIUM: 'medium',
+  LARGE: 'large',
+};
+
+const SearchInput = ({
+  onClear,
+  value,
+  width,
+  size = 'medium',
+  className,
+  style,
+  ...props
+}) => {
+  const handleClick = (e) => {
+    e.preventDefault();
+    onClear();
+  };
+  const handleKeyDown = (e) => {
+    if (e.key === 'Escape') onClear();
+  };
+
+  return (
+    <StyledContainer
+      width={width}
+      className={className}
+      style={style}
+      size={size}
+    >
+      <StyledIcon
+        css={css`
+          position: absolute;
+          left: 1rem;
+          top: 50%;
+          transform: translateY(-50%);
+        `}
+        name={Icon.TYPE.SEARCH}
+        size={size}
```

Comment:
Since you're setting the size of the icon in the `StyledIcon` component below, you can omit this.

Reply:
hmm if I remove it I lose the varied sizing, though I'm not entirely sure why

Reply:
Oh duh, you're using that prop within `StyledIcon`. When I wrote this, somehow my brain thought you were using `Icon` directly. Ignore this! Carry on 😆